webapp.py

```
@@ -0,0 +1,285 @@
+    """ Start the script if it hasn't been started already. If it has, do nothing, returning a failure response. """
+    if script_is_running():
+        return build_response(
+            success=False,
+            statusText="A connection to the League client has already been established.",
+            status=409,
+        )
+
+    try:
+        state.connection = c.Connection()
+        run_on_thread(main_loop.main_loop, state.connection)
+    except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)